cmd/downloads/config/downloads_config_test.go

```
@@ -150,13 +151,14 @@ func TestCreateArchivesInBackground(t *testing.T) {
-		tarPath := base + ".tar"
+		tarPath := base + ".tar.gz"
 		if _, err := os.Stat(tarPath); err != nil {
 			t.Errorf("tar archive not found at %s: %v", tarPath, err)
 			continue
 		}
 		f, _ := os.Open(tarPath)
-		tr := tar.NewReader(f)
+		gzw, _ := gzip.NewReader(f)
+		tr := tar.NewReader(gzw)
 		hdr, err := tr.Next()
 		f.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `f.Close` is not checked

(errcheck)